### PR TITLE
Md sa2 28 24 Kazimirov

### PR DESCRIPTION
**Pull request by Kirilllka1993**

### CODE REVIEW

File: Kirill_Kazimirov/02.Git.Local/README.md [hunk 1]
```diff
@@ -0,0 +1,135 @@
+#Homework Assignment 1: Initializing a Local Repository
```
Comment (pluhin) [inline Kirill_Kazimirov/02.Git.Local/README.md:1]: You need to add white-space after #

please correct all headers

File: Kirill_Kazimirov/02.Git.Local/README.md [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+    7  cd /home/kirill/courses/devops/sa.it-academy.by/Kirill_Kazimirov/02.Git.Local
```
Comment (pluhin) [inline Kirill_Kazimirov/02.Git.Local/README.md:2]: Please use code block for history sections 
https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#quoting-code

**Review by pluhin — CHANGES_REQUESTED** [review]

File: Kirill_Kazimirov/02.Git.Local/README.md (hunk 1 above)
Comment (Kirilllka1993) [reply Kirill_Kazimirov/02.Git.Local/README.md]: Fix it

File: Kirill_Kazimirov/02.Git.Local/README.md (hunk 2 above)
Comment (Kirilllka1993) [reply Kirill_Kazimirov/02.Git.Local/README.md:3]: Fix it

**Review by pluhin — APPROVED** [review]

**Pull request merged**